File: 38-Java/src/Roulotte.java
public class Roulotte extends Veicolo {
    private int peso;
    private String veranda;
    
    
    public Roulotte(int codice, int postiLetto, String tipo, String marca, float lunghezza, float larghezza,
            float costoGiornaliero, int peso, String veranda) {
        super(codice, postiLetto, tipo, marca, lunghezza, larghezza, costoGiornaliero);
        this.peso = peso;
        this.veranda = veranda;
    }


    public String toString(){
        return super.toString() + "\t" + peso + "\t" + veranda + "\t-\t" + costoGiornaliero;
    }
}

File: 38-Python/Roulotte.py
from Veicolo import Veicolo

class Roulotte(Veicolo):
    def __init__(self, codice, marca, larghezza, lunghezza, costo, posti, peso, veranda) -> None:
        super().__init__(codice, marca, larghezza, lunghezza, costo, posti)
        self._peso = peso
        self._veranda = veranda

    def __str__(self) -> str:
        return "roulotte\t" + super().__str__() + "\t" + self._veranda + "\t-\t" + str(self._costo)